appveyor.yml

```diff
@@ -18,45 +26,98 @@ matrix:
 #   - x86
 #   - x64
 
-# Install scripts. (runs after repo cloning)
-install:
-  # Get the latest stable version of Node.js or io.js
-  - ps: Install-Product node $env:nodejs_version
-
-  # install modules
-  - ps: >-
-      if ($env:nodejs_version -eq "4") {
-        npm install -g npm@3;
-      }
-      if ($env:nodejs_version -in @("8", "10", "12")) {
-        npm install -g npm@6.10.3;
-      }
-  - npm install
-
-  # fix symlinks
-  - cmd: git config core.symlinks true
-  - cmd: git reset --hard
-
-  # todo: learn how to do this for all .\resolvers\* on Windows
-  - cd .\resolvers\webpack && npm install && cd ..\..
-  - cd .\resolvers\node && npm install && cd ..\..
-
-# Post-install test scripts.
-test_script:
-
-  # Output useful info for debugging.
-  - node --version
-  - npm --version
-
-  # core tests
-  - npm run tests-only
-
-  # resolver tests
-  - cd .\resolvers\webpack && npm test && cd ..\..
-  - cd .\resolvers\node && npm test && cd ..\..
-
-on_success:
-  - npm run coveralls
+for:
+-
+  matrix:
+    only:
+      - configuration: Native
+  # Install scripts. (runs after repo cloning)
+  install:
+    # Get the latest stable version of Node.js or io.js
+    - ps: Install-Product node $env:nodejs_version
+
+    # install modules
+    - ps: >-
+        if ($env:nodejs_version -eq "4") {
+          npm install -g npm@3;
+        }
+        if ($env:nodejs_version -in @("8", "10", "12")) {
+          npm install -g npm@6.10.3;
+        }
+    - npm install
+
+    # fix symlinks
+    - cmd: git config core.symlinks true
+    - cmd: git reset --hard
+
+    # todo: learn how to do this for all .\resolvers\* on Windows
+    - cd .\resolvers\webpack && npm install && cd ..\..
+    - cd .\resolvers\node && npm install && cd ..\..
+
+  # Post-install test scripts.
+  test_script:
+
+    # Output useful info for debugging.
+    - node --version
+    - npm --version
+
+    # core tests
+    - npm run tests-only
+
+    # resolver tests
+    - cd .\resolvers\webpack && npm test && cd ..\..
+    - cd .\resolvers\node && npm test && cd ..\..
+
+  on_success:
+    - npm run coveralls
+-
+  matrix:
+    only:
+      - configuration: WSL
+  # Install scripts. (runs after repo cloning)
+  install:
+    # Get the latest stable version of Node.js or io.js
+    - ps: $env:WSL_ENV += ":nodejs_version"
+    - ps: >-
+      wsl curl -sL 'https://deb.nodesource.com/setup_${nodejs_version$}.x' `| sudo -E bash -
+      wsl sudo apt install -y nodejs
```

Comment:
this line seems to trigger a parsing error?

Reply:
Yeah - I indented the code incorrectly.
I just fixed it... no idea whether this WSL thing actually will work but we'll see 😅